umap/tests/test_views.py

```diff
@@ -157,6 +158,7 @@ def test_stats_empty(client):
         "maps_count": 0,
         "users_active_last_week_count": 0,
         "users_count": 0,
+        "version": VERSION,
```

Comment:
At least we are testing the existence of the key 😅

Reply:
How would you test that without needing to update the value at each version otherwise ? :)

Reply:
You're right, monkeypatching would be a lie anyway 👍